cmake/FindGTS.cmake

```diff
@@ -65,7 +65,14 @@ else()
   endif()
 
   # 2.1 Need glib library
+  find_path(GLIB_INCLUDE_DIR
+    NAMES glib.h
+    PATH_SUFFIXES glib-2.0)
+  find_path(GLIBCONFIG_INCLUDE_DIR
+    NAMES glibconfig.h
+    PATH_SUFFIXES lib/glib-2.0/include)
   find_library(GLIB_LIBRARY glib-2.0)
+  list(APPEND GTS_INCLUDE_DIRS "${GLIB_INCLUDE_DIR}" "${GLIBCONFIG_INCLUDE_DIR}")
```

Comment:
> the worst that can happen is that the strings `GLIB_INCLUDE_DIR-NOTFOUND` and `GLIBCONFIG_INCLUDE_DIR-NOTFOUND` get appended to `GTS_INCLUDE_DIRS`. I think that adding invalid include paths shouldn't break anything.

I believe you're correct that invalid include paths shouldn't break anything (they should be "ignored"), but just to be safe, what if we added a check that only adds the include directories if they've been found? We could do something like check to see if `GLIB_INCLUDE_DIR` or `GLIB_CONFIG_INCLUDE_DIR` contain `NOTFOUND` before calling `list(APPEND ...)`.

Reply:
If believe we could do something like:

```cmake
if (GLIB_INCLUDE_DIR MATCHES "NOTFOUND")
  list(APPEND GTS_INCLUDE_DIRS "${GLIB_INCLUDE_DIR}"
endif()
if (GLIBCONFIG_INCLUDE_DIR MATCHES "NOTFOUND")
  list(APPEND GTS_INCLUDE_DIRS "${GLIBCONFIG_INCLUDE_DIR}"
endif()
```

(https://cmake.org/cmake/help/latest/command/if.html#matches)

Reply:
Good idea, I saw that just doing `if (GLIB_INCLUDE_DIR)` is enough. See 4c9a899

Reply:
Nice, I didn't realize we could do that - looks good!